CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
 # CHANGELOG
 
 ## XX.XX.XX - 20XX-XX-XX
+
+## 20.41.1 - 2024-05-28
 * [FIXED][8538](https://github.com/stripe/stripe-android/pull/8538) Change Autocomplete Screen "Enter address manually" button background to prevent unexpected changes to user set colors
+* [FIXED][8537](https://github.com/stripe/stripe-android/pull/8537) Fixed cardscan issue #7915 where crashes were happening on some devices with odd camera vs. screen resolutions
```

Comment:
These notes are missing the separation by product. Can you add `PaymentSheet` and `CardScan` headers?

It should also be `20.44.1`, not `20.41.1`.

Also tweaked the text a little.

```suggestion
## 20.44.1 - 2024-05-28

### PaymentSheet
* [FIXED][8538](https://github.com/stripe/stripe-android/pull/8538) Change the autocomplete screen’s "Enter address manually" button background to prevent unexpected changes to user-defined colors

### CardScan
* [FIXED][8537](https://github.com/stripe/stripe-android/pull/8537) Fix a crash that happened on some devices with odd camera-to-screen ratios
```

Reply:
what a typo! I'll close this PR and re-open with a new one with the correct version.